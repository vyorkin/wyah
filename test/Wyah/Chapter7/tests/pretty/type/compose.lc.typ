(TVar (TV "c") :-> TVar (TV "b")) :-> ((TVar (TV "a") :-> TVar (TV "c")) :-> (TVar (TV "a") :-> TVar (TV "b")))
