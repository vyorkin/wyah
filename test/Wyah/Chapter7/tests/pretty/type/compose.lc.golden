(c -> b) -> (a -> c) -> a -> b